app/src/main/java/com/github/se/gomeet/ui/mainscreens/Events.kt

```diff
@@ -13,19 +13,29 @@ import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.offset
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
 import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
```

Comment:
use material3 instead

Reply:
![Capture d'écran 2024-04-21 131557](https://github.com/SwEnt-Project-G18/SwEntApp/assets/125990940/8d2821af-e6e6-43b3-b54a-e33b8778384d)
I searched on google but I can't find any material3 library for icons

Reply:
my bad, you're right